index.html

```diff
@@ -43,6 +43,142 @@ <h1>
+            </div>
+          </header>
+          <p class="project-summary">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="project-tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <a href="#" class="btn">See Project</a>
+        </div>
+      </article>
+    </div>
+    <section class="about" id="about">
+      <section class="headline">
+        <h2> About <br> Myself </h2>
+        <p class="headline-text">
+          I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <div class="social-media">
+          <p class="supporting-text">LET’S CONNECT</p>
+        </div>
+        <ul class="media-icons">
+          <li><a href="https://github.com/razvanso"><img src="images/GitHub.svg" alt="GitHub"></a></li>
+          <li><a href="https://www.linkedin.com/in/razvan-rio/"><img src="images/Linkedin.svg" alt="Linkedin"></a></li>
+          <li><a href="#"><img src="images/Vector.svg" alt="Vector"></a></li>
+          <li><a href="https://twitter.com/RioRazvan"><img src="images/Twitter.svg" alt="Twitter"></a></li>
+          <li><a href="#"><img src="images/Medium.svg" alt="Medium"></a></li>
+        </ul>
+        <a href="#" class="btn">Get my resume</a>
+        <section class="skill-list">
+          <input id="languages" class="toggle" type="checkbox" checked>
+          <label for="languages" class="toggle-label">Languages</label>
+          <ul class="collapsible-category">
+            <li><img src="images/works/JS.png" alt="JavaScript logo">JavaScript</li>
+            <li><img src="images/works/HTML.png" alt="HTML logo">HTML</li>
+            <li><img src="images/works/CSS.png" alt="CSS logo">CSS</li>
+          </ul>
+        </section>
+        <section class="skill-list">
+          <input id="frameworks" class="toggle" type="checkbox">
+          <label for="frameworks" class="toggle-label">Frameworks</label>
+          <ul class="collapsible-category">
+            <li><img src="images/works/HTML.png" alt="HTML logo">HTML</li>
+            <li><img src="images/works/CSS.png" alt="CSS logo">CSS</li>
+            <li><img src="images/works/JS.png" alt="JavaScript logo">JavaScript</li>
+          </ul>
+        </section>
+        <section class="skill-list">
+          <input id="skills" class="toggle" type="checkbox">
+          <label for="skills" class="toggle-label">Skills</label>
+          <ul class="collapsible-category">
+            <li><img src="images/works/CSS.png" alt="CSS logo">CSS</li>
+            <li><img src="images/works/JS.png" alt="JavaScript logo">JavaScript</li>
+            <li><img src="images/works/HTML.png" alt="HTML logo">HTML</li>
+          </ul>
+        </section>
```

Comment:
- [ ] Please wrap these three `.skill-list` sections using a `ul` and place each of them inside a `li` tag. the `.skill-list`s are a list of different skill sections so it might make more sense to have them inside a `ul`. I hope this makes sense.

Reply:
Great feedback! Thanks